test/test_jit.py

```diff
@@ -2052,6 +2052,22 @@ def forward(self, x, weight, bias):
         res = m(x, weight, bias)
         self.assertEqual(res, ref_res)
 
+    def test_finalize(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.fc = torch.nn.Linear(5, 5).float()
+
+            def forward(self, x):
+                return self.fc(x)
+
+        data = [(torch.rand((1, 5), dtype=torch.float), torch.randint(0, 1, (1,), dtype=torch.long)) for _ in range(2)]
+        qconfig_dict = {'': default_qconfig}
+        model = torch.jit.script(M()).eval()
+        model = quantize_script(model, qconfig_dict, _test_only_eval_fn, [data], inplace=False)
+        FileCheck().check("quantized::linear") \
+                   .run(model.graph)
+
```

Comment:
Please add a test to verify that pre_pack op is present in the graph so that the pattern match is tested.

Reply:
quantized::linear is looking for prerpack ops, so when we see quantized::linear in the graph we can be sure that prepacked ops are there